lib/diagnostic.rb

```diff
@@ -78,17 +93,24 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = I am a HouseCat, and I go 'meow'
```

Comment:
This will be interpreted literally and cause an error.  This should be wrapped in quotation marks.